proto/tendermint/types/evidence.proto

```diff
@@ -28,5 +28,4 @@ message Evidence {
 // EvidenceData contains any evidence of malicious wrong-doing by validators
 message EvidenceData {
   repeated Evidence evidence = 1 [(gogoproto.nullable) = false];
-  bytes             hash     = 2;
```

Comment:
Will this break anything?

Reply:
I don't think so it just changes the size of the struct but we still calculate the hash

Reply:
Is this type passed over ABCI?

Reply:
No it's broken down to the individual evidence

Reply:
👍 in favor of removing (we can always calculate it on the fly).